taglib/ape/apefile.cpp

```diff
@@ -52,28 +52,18 @@ namespace
 class APE::File::FilePrivate
 {
 public:
-  FilePrivate() = default;
-  ~FilePrivate()
-  {
-    delete ID3v2Header;
-    delete properties;
-  }
-
-  FilePrivate(const FilePrivate &) = delete;
-  FilePrivate &operator=(const FilePrivate &) = delete;
```

Comment:
Why should these go away?

Reply:
The reason for them is to fix a clang-tidy warning regarding a non default constructor.

Reply:
Actually now that I think about it because of unique_ptr, it’s not copyable anyway.